task/buildah/0.5/buildah.yaml

```diff
@@ -56,6 +56,13 @@ spec:
   - name: source
   - name: sslcertdir
     optional: true
+  - name: secrets
+    optional: true
+    description: >-
```

Comment:
thank you for the PR @jimmyjones2, can you please document this in the readme as well?

Reply:
Done